packages/legacy/handlers/grpc/src/index.ts

```diff
@@ -466,6 +466,13 @@ export default class GrpcHandler implements MeshHandler {
               if (enumSerializedValue) {
                 const serializedValue = JSON.parse(enumSerializedValue);
                 value.value = serializedValue;
+                let valueLookup = (type as any)._valueLookup;
```

Comment:
Do you think we can add some tests with enums to prevent future regressions?

Reply:
Ok I'll add tests after! Thanks for the PR!

Reply:
Thanks for being so responsive!